sdcm/results_analyze/__init__.py

```diff
@@ -636,10 +638,17 @@ def check_regression(self, test_id, is_gce=False, email_subject_postfix=None, us
         }
         self.log.debug('Regression analysis:')
         self.log.debug(PP.pformat(results))
-        test_name = full_test_name.split('.')[-1]  # Example: longevity_test.py:LongevityTest.test_custom_time
+        test_name = full_test_name.split('.', 1)[1]  # Example: longevity_test.py:LongevityTest.test_custom_time
```

Comment:
here you are taking IMHO the wrong list's item:
```
full_test_name.split('.', 1)[1]
'py:LongevityTest.test_custom_time'
full_test_name.split('.', 1)[0]
'longevity_test'
```

Reply:
This is the fullname in my case:
ycsb_performance_regression_test.YCSBPerformanceRegression100MRecordsTest.test_latency_read_write_modify

I think this comment isn't up-to-date...

Reply:
Here more example from recent emails:

performance_regression_test.PerformanceRegressionTest.test_latency_write_with_nemesis

I'll update the comment to avoid confusion.

I'll I wanted here that the email title would have the class to avoid confusion, since test name are quite similar between different use cases.

Reply:
fixed the comment